cheri-c-programming.tex

```diff
@@ -896,7 +896,21 @@ \subsubsection{Out-of-bounds pointers}
 
 \note{I feel like this section wants a reference to CHERI Concentrate?}{nwf}
 
-The capability compression model exploits redundancy between the pointer and
+ISO C permits pointers to go only one byte beyond their original
```

Comment:
I tried to use `address` when talking about the specific address component of a capability and use `pointer` to mean a C language pointer or the capability as a whole when used as a pointer.  I think some of the existing language here and elsewhere might use `pointer` to mean just the address?